tests/test-exoskeleton-add-rule.php

```diff
@@ -0,0 +1,279 @@
+<?php
+/**
+ * Class ExoskeletonTest
+ *
+ * @package Exoskeleton
+ */
+
+/**
+ * Sample test case.
+ */
+class ExoskeletonAddRuleTest extends WP_UnitTestCase {
+
+
+	/**
+	 * Three rule definitions
+	 *
+	 * @var array Three valid rule definitions
+	 */
+	protected static $three_valid_rules;
```

Comment:
per my review this is no longer utilized. If that is true we should clean it up/remove.